applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -1693,6 +1786,104 @@ void BaseSolidElement::CalculateAndAddExtForceContribution(
 /***********************************************************************************/
 /***********************************************************************************/
 
+void BaseSolidElement::CalculateLumpedMassVector(VectorType& rMassVector)
+{
+    KRATOS_TRY;
+
+    const auto& r_geom = GetGeometry();
+    const auto& r_prop = GetProperties();
+    const SizeType dimension = r_geom.WorkingSpaceDimension();
+    const SizeType number_of_nodes = r_geom.size();
+    const SizeType mat_size = dimension * number_of_nodes;
+
+    // Clear matrix
+    if (rMassVector.size() != mat_size)
+        rMassVector.resize( mat_size, mat_size );
```

Comment:
is this correct?  It is a vector not a matrix

Reply:
Merde

Reply:
Solved, I wanted to put false